src/containers/AdminCampaignStats.jsx

```diff
@@ -272,11 +261,26 @@ class AdminCampaignStats extends React.Component {
                         <RaisedButton
                           {...dataTest("copyCampaign")}
                           label="Copy Campaign"
-                          onTouchTap={async () =>
-                            await this.props.mutations.copyCampaign(
+                          onTouchTap={async () => {
+                            let result = await this.props.mutations.copyCampaign(
                               this.props.params.campaignId
-                            )
-                          }
+                            );
+                            if (
+                              window.confirm(
```

Comment:
I support adding feedback on-completion but it should not be through `window.confirm()` which forces an interaction.  Many campaigners will click this button 10 times from a campaign template so they can then go edit them all at once -- they shouldn't be interrupted with a popup window -- please add a `setState()` call that updates the render with a SnackBar link to the new campaign.

Reply:
Okay, the `Snackbar` is in, and I added what I believe are the appropriate tests under `__test__/backend.test.js`.